db/email_templates/account_deactivated.html.erb

```diff
@@ -0,0 +1,794 @@
+                  <td valign="top" class="bodyContainer">
+                    <table border="0" cellpadding="0" cellspacing="0" width="100%" class="mcnTextBlock">
+                      <tbody class="mcnTextBlockOuter">
+                      <tr>
+                        <td valign="top" class="mcnTextBlockInner">
+                          <table align="left" border="0" cellpadding="0" cellspacing="0" width="600" class="mcnTextContentContainer">
+                            <tbody>
+                            <tr>
+                              <td valign="top" class="mcnTextContent" style="padding-top:9px; padding-right: 18px; padding-bottom: 9px; padding-left: 18px;">
+                              </td>
+                            </tr>
+                            </tbody>
+                          </table>
+                        </td>
+                      </tr>
+                      </tbody>
+                    </table>
+                  </td>
+                </tr>
+              </table>
+              <!-- // END BODY -->
+            </td>
+          </tr>
+          <tr>
+            <td align="center" valign="top">
+              <!-- BEGIN FOOTER // -->
+              <table border="0" cellpadding="0" cellspacing="0" width="600" id="templateFooter">
+                <tr>
+                  <td valign="top" class="footerContainer" style="padding-bottom:9px;">
+                    <table border="0" cellpadding="0" cellspacing="0" width="100%" class="mcnTextBlock" style="min-width:100%;">
+                      <tbody class="mcnTextBlockOuter">
+                      <tr>
+                        <td valign="top" class="mcnTextBlockInner" style="padding-top:9px;">
+                          <!--[if mso]>
+                          <table align="left" border="0" cellspacing="0" cellpadding="0" width="100%" style="width:100%;">
+                            <tr>
+                          <![endif]-->
+                          <!--[if mso]>
+                          <td valign="top" width="600" style="width:600px;">
+                          <![endif]-->
+                          <table align="left" border="0" cellpadding="0" cellspacing="0" style="max-width:100%; min-width:100%;" width="100%" class="mcnTextContentContainer">
+                            <tbody>
+                            <tr>
+                              <td valign="top" class="mcnTextContent" style="padding-top:0; padding-right:18px; padding-bottom:9px; padding-left:18px;">
+                                <div style="text-align: center;">
+                                  <div style="line-height: 20.8px; text-align: center;">
+                                    <span style="font-size:12px">
+                                      <span style="font-family:helvetica neue,helvetica,arial,verdana,sans-serif">
+                                        <span style="line-height:20.8px">You received this message because you have an account&nbsp;on&nbsp;
```

Comment:
I just eyeballed the HTML, but this jumped out at me as inconsistent with the messaging (maybe copy-pasta'd from your other template?)
```
                                        <span style="line-height:20.8px">You received this message because you have an account&nbsp;on&nbsp;
                                          <a href="https://search.gov" target="_blank">Search.gov</a> that is set to expire soon.&nbsp;
```

Reply:
ah yes looks like the mockups had that incorrectly.  I will change the wording to You received this message because you have an account that is deactivated.

Reply:
<img width="758" alt="Screen Shot 2020-01-24 at 6 58 54 PM" src="https://user-images.githubusercontent.com/7727136/73112246-9f530700-3edb-11ea-8a6d-9e82c7701148.png">